src/disk.py

```diff
@@ -53,8 +61,9 @@ def main(args):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("-t", "--total", action="store_true", default=False)
-    parser.add_argument("-f", "--free", action="store_true", default=False)
-    parser.add_argument("-p", "--path", type=str, default=_get_default_path())
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument("-t", "--total", action="store_true", default=False)
+    group.add_argument("-f", "--free", action="store_true", default=False)
+    group.add_argument("-p", "--path", type=str, default=None)
```

Comment:
The `--path` argument probably should not be in the mutually exclusive group, right?

Reply:
You're 100% correct, it has been fixed!